### DISCUSSION TITLE
Using ArgoCD status badge in README.md files

### DISCUSSION THREAD

**Discussion by jyotisingh-deloitte** [argoproj/argo-cd#7225] [Q&A] [answered]
Hi Team,

I am trying to use ArgoCD status badge in the Readme file and following the instructions as per this document: https://argo-cd.readthedocs.io/en/stable/user-guide/status-badge/

I have successfully enabled the badge for my application and I can also see the badge under application details in argocd UI.
But URL is not working properly when used in Readme file and I cant see the badge.

PS: I have VPN setup enabled to access applications deployed in argocd. So, I am thinking that might be the issue. 

My question here is, is there any way to display this badge by getting past the VPN, as in the documentation it says that no authentication is required to access the ArgoCD status badge.

**Comment by wanghong230** [comment, marked as answer]
The badge works in a way your browser to access the URL. If your browser can access the API, it should be working.

VPN and authentication are totally different things. VPN is about connectivity.

**Reply by jyotisingh-deloitte** [reply]
ok that's right.
Yeah, then I guess as for my applications in ArgoCD I need to be on VPN to access it, similarly I need to be on VPN to access the Status badge API as well ?

**Reply by wanghong230** [reply]
Yes, your understanding is right. There is no magic here.
